src/lib/UserSetting/UserSettingGroup.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\User\UserSetting;
+
+use Ibexa\Contracts\Core\Repository\Values\ValueObject;
+
+class UserSettingGroup extends ValueObject
```

Comment:
Would be nice if you could add a constructor and get/set methods or at least `@property-*` annotations so we can have nice type hinting for this class.